active-directory-wpf-msgraph-v2/MainWindow.xaml.cs

```diff
@@ -37,9 +42,20 @@ private async void CallGraphButton_Click(object sender, RoutedEventArgs e)
             ResultText.Text = string.Empty;
             TokenInfoText.Text = string.Empty;
 
-            var accounts = await app.GetAccountsAsync();
-            var firstAccount = accounts.FirstOrDefault();
+            IAccount firstAccount;
 
+            // WAM will always get an account in the cache. So if we want
+            // to have a chance to select the accounts interactively, we need to
+            // force the non-account
+            if (howToSignIn.SelectedIndex == 1)
+            {
+                firstAccount = null;
+            }
+            else
```

Comment:
if (howToSingIn.SelectedIndex!=1) { ... }

Reply:
not for the moment (until null uses the default accounts. This was not the case in 4.20 yet